doc/interop-test-descriptions.md

```diff
@@ -678,8 +677,12 @@ canonical form of the authenticated source. The canonical form is dependent on
 the authentication method, but is likely to be a base 10 integer identifier or
 an email address.
 
+If a SimpleRequest has fill_oauth_scope=true and that request was successfully authenticated via OAuth, then the SimpleResponse should have oauth_scope filled with the scope of the method being invoked.
```

Comment:
This section is a partial duplicate of the paragraph above (that I said to split out). The text from both should be combined.